html-react-parser with React 19

html-react-parser with React 19

I am experiencing an issue with the html-react-parser package when using it with React 19. I’ve encountered compatibility problems which prevent the package from installing.

Version: 5.1.13

React Version: 19.0.0-rc (release candidate version)

Error Details: When attempting to install or use html-react-parser with React 19, npm reports a peer dependency conflict. The specific error message is:

```
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: nextjs@0.1.0
npm error Found: react@19.0.0-rc-fb9a90fa48-20240614
npm error node_modules/react
npm error   react@"^19.0.0-rc-fb9a90fa48-20240614" from the root project
npm error
npm error Could not resolve dependency:
npm error peer react@"0.14 || 15 || 16 || 17 || 18" from html-react-parser@5.1.13
npm error node_modules/html-react-parser
npm error   html-react-parser@"*" from the root project
```

--- a/package.json
+++ b/package.json
@@ -91,8 +91,8 @@
     "typescript": "5.7.2"
   },
   "peerDependencies": {
-    "@types/react": "0.14 || 15 || 16 || 17 || 18",
-    "react": "0.14 || 15 || 16 || 17 || 18"
+    "@types/react": "0.14 || 15 || 16 || 17 || 18 || 19",
+    "react": "0.14 || 15 || 16 || 17 || 18 || 19"
   },
   "peerDependenciesMeta": {
     "@types/react": {

